src/components/molecules/timeline/timeline.tsx

```diff
@@ -18,14 +18,14 @@ const formatDate = (date: string): string => new Date(date).toLocaleString('en-G
 
 export const Timeline = ({ events }: TimelineProps): JSX.Element => (
   <div className="review-timeline">
-    <dl className="review-timeline__list">
+    <dl className="review-timeline__list" aria-label="This is the timeline of this Reviewed Preprint">
```

Comment:
As discussed, let's make this `aria-label` value a parameter, could be required. We can add an example in storybook.